tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -233,3 +313,66 @@ def test_harvesting_arxiv_workflow_manual_accepted(
         obj = eng.processed_objects[0]
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_workflows_change_document_type_article(workflow_app, insert_journals_in_db, workflow):
```

Comment:
the convention is to have `test_{function_name}_{extra_details}`, so something like `test_set_document_type_and_refereed_all_proceedings_none_refereed`. This applies to all your tests.